apps/ledger-live-mobile/src/screens/SyncOnboarding/SyncOnboardingCompanion.tsx

```diff
@@ -369,17 +370,22 @@ export const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = (
     if (deviceOnboardingState?.isOnboarded && !seededDeviceHandled.current) {
       if (deviceOnboardingState?.currentOnboardingStep === DeviceOnboardingStep.Ready) {
         // device was just seeded
-        setCompanionStepKey(CompanionStepKey.Backup);
+        setCompanionStepKey(
+          recoverUpsellRedirection ? CompanionStepKey.Apps : CompanionStepKey.Backup,
```

Comment:
[SHOULD] use `recoverUpsellRedirection?.enabled`.
[SHOULD] same comment as for LLM, `const firstStepKeyAfterSeeding = recoverUpsellRedirection?.enabled ? CompanionStepKey.Apps : CompanionStepKey.Backup`